automation/README.md

```diff
@@ -0,0 +1,83 @@
+- def is_connected
+  - Checks if connection is still active. Doesn't take any arguments returns a bool. True if connection is still good, false otherwise.
+- def decision_mapping(self, mgs: str) -> Tuple[str, List[str]]
+  - Translates the model pipeline result to an automation *action*. Takes as an argument a string representation of gadget_core pipeline message. Returns a tuple, a string representing the *action* and a list of strings representing any metadata associated with the action.
+- def send_action(self, action: str, aux_info: List[str]) -> bool:
+  - Sends the action to the automation machine. It takes the result of decision_mapping as its arguments, a string representing the action and a list of strings representing any metadata. It returns a bool. True if the action was sent, False otherwise
+
+## Configs JSON File
+
+In addition to custom code, the automation server expects a JSON file the defines any configs to be stored in the database. The JSON file must include at least the keys *configs_def* and *configs_with_handlers* which both go to a list of objects.
+
+    {
+        "configs_def": [
+        
+        ],
+        "configs_with_handlers": [
+        
+        ]
+    }
+
+To define a config, add to the configs_def list an object with two key value pairs, *name* and *default_value* as the keys. As an example:
+
+    "configs_def": [
+         {
+            "name": "ip_address",
+            "default_value": "10.10.11.110"
+        }
+    ],
+
+To define a config with a handler, add to the configs_with_handlers list an object with three key value pairs, *name*, *default_value* and *handler* as the keys. As an example:
+
+    "configs_with_handlers": [
+            {
+                "name": "line_speed",
+                "default_value": 5,
+                "handler": "line_speed_handler"
+            }
+    }
+
+The value associated with the key *handler* must be a method defined in the automation class. The method must take the config value as an argument and not return anything. The automation server will call that handler whenever the config is updated in the database. The complete JSON file from the examples would look like:
+
+    {
+        "configs_def": [
+            {   
+                "name": "ip_address",
+                "default_value": "10.10.11.110"
+            }
+         ],
+        "configs_with_handlers": [
+            {
+                "name": "line_speed",
+                "default_value": 5,
+                "handler": "line_speed_handler"
+            }
+        }
+    }
+
+## Dockerfile
+
+Because this is a custom container a custom dockerfile must be created too. By default the automation service definition inside the inspection.docker-compose.yaml looks for a file named automation.dockerfile. If the file name is changed the docker-compose must be updated.
```

Comment:
``Because this is a custom container a custom dockerfile must be created too.``
need a comma.

Reply:
``If the file name is changed the docker-compose must be updated``
need a comma